CRM/Batch/Form/Entry.php

```diff
@@ -669,28 +658,6 @@ private function processMembership(&$params) {
 
         $membershipTypeId = $value['membership_type_id'] = $value['membership_type'][1];
 
-        foreach ($dateTypes as $dateField => $dateVariable) {
```

Comment:
moved down to the single membership section for clarity, since this does not relate to renewal & causes confusion